auditors/nonroot/fixtures/run-as-user-psc-1-run-as-non-root-psc-true.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod
+  labels:
+    name: pod
+  namespace: run-as-user-psc-1-run-as-non-root-psc-true
+spec:
+  securityContext:
+    runAsUser: 1
+    runAsNonRoot: false
```

Comment:
```suggestion
    runAsNonRoot: true
```

Reply:
The goals of these tests (the one with `true` and this one with `false`) was to ensure that the `runAsRoot`, whatever its value is, doesn't have any incidence on the result if `runAsUser` is used. But maybe this test is overkill and we can delete it?

Reply:
They are both set to false currently. The name of this file has `run-as-non-root-psc-true` so I assumed it was intended to be `true` 😉